metrics/metrics.go

```diff
@@ -39,11 +39,21 @@ var (
 		},
 		[]string{"type"},
 	)
+	// ScrappedErrorTotal is a Prometheus gauge to track the number of times that an scrapped aggregated error is produced.
+	ScrappedAggregatedErrorTotal = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: PrometheusNamespace,
+			Name:      "scrapped_aggregated_error_total",
```

Comment:
how about `error_occurrences_total `?

Description: Total number of occurrences per service, error type and severity.

Reply:
Let's remove the suffix _`total` if it's a gauge. See naming conventions for Prometheus https://www.robustperception.io/on-the-naming-of-things